.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=./lib/:${PYTHONPATH}
```

Comment:
Ugh; that's the problem with switching from a branch with a `.gitignore` change!